src/material/tooltip/tooltip.ts

```diff
@@ -202,14 +205,17 @@ export class MatTooltip implements OnDestroy, AfterViewInit {
   @Input('matTooltip')
   get message() { return this._message; }
   set message(value: string) {
-    this._ariaDescriber.removeDescription(this._elementRef.nativeElement, this._message);
+    if (this._message) {
```

Comment:
Rather than checking `message` here, could `AriaDescriber` just check this and no-op immediately?

Reply:
Updating in #19777